koku/api/report/test/aws/test_queries.py

```diff
@@ -1581,6 +1592,7 @@ def test_execute_query_with_tag_filter(self):
         result = data_totals.get("cost", {}).get("total", {}).get("value")
         self.assertEqual(result, totals["cost"])
 
+    @skip("Fix after currency support goes in")
```

Comment:
Do these skipped tests have associate JIRA tickets?

Reply:
This is the only one that does not; however, I did already chat with Ash about this test. Ash already has a potential fix inside of the currency support pr, and I will be helping out over there with a few things after this goes in so I am pretty much gonna merge and start immediately working on it over there to prevent conflicts.